recipes/ffnet/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "ffnet" %}
+{% set version = "0.8.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "1aaf35158239cd1e6f63504e884683abac87c77919af36cbefbf1169fed5d1a2" %}
```

Comment:
it looks like the `sha256` should be:

4d7befca0247108fc05fb7ebe7cdcff78eb642c4ab8c93215f592dcfed0f29f6

I think you were grabbing the value for the `.zip` copy of the source.